package.json

```diff
@@ -28,26 +28,30 @@
     "@react-navigation/native": "^5.0.0",
     "@react-navigation/stack": "^5.0.0",
     "@rootstrap/redux-tools": "^0.4.2",
+    "add": "^2.0.6",
     "axios": "^0.19.2",
     "eslint-plugin-prettier": "3.1.0",
     "humps": "2.0.1",
     "immer": "5.0.0",
     "lodash": "4.17.15",
     "prop-types": "15.7.2",
-    "react": "16.9.0",
-    "react-native": "0.61.5",
+    "react": "16.11.0",
+    "react-native": "0.62.2",
     "react-native-config": "0.12.0",
+    "react-native-flipper": "^0.50.0",
```

Comment:
Should this library be a devDependency?

Reply:
In the instructions it didn't specify that you should put it as a dev dependency, I tried anyway and it seems to be working.